### DISCUSSION TITLE
HOW TO UPDATE FLOWISE

### DISCUSSION THREAD

**Discussion by giovanninino** [FlowiseAI/Flowise#1297] [General]
Hi guys I hope you are well! Could please some one tell me how update flowise? Thanks in advance

**Comment by yavisht** [comment]
@giovanninino 

If you are running it via render or railway, simply redeploy the code. It should pull the latest copy and publish it.

PS: make sure you have a persistent volume setup to avoid wiping your existing data

If you are using it on localhost, just run git pull on the main branch and run it. 

```
cd the-flowise-repo-folder
git checkout main
git pull origin main
```

Further reading:
https://docs.flowiseai.com/deployment
